src/klein/_form.py

```diff
@@ -393,9 +396,8 @@ def glue(self) -> Iterable[Tag]:
 
         @return: some HTML elements in the form of renderable objects for
             L{twisted.web.template}
-        @rtype: L{twisted.web.template.Tag}, or L{list} thereof.
         """
-        return self._fieldForCSRF().asTags()
+        return list(self._fieldForCSRF().asTags())
```

Comment:
This suggests that the type hint in Twisted is too narrow?

Reply:
OK, looking at the code, I think not. So now I'm wondering if copying to a new list is necessary here…

Reply:
OK will worry about this later